js/components/Checkbox.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
```

Comment:
Well if we're going to pull this code into our project, might as well make look like the rest of our ES6 JavaScript syntax